app/src/main/java/com/swent/assos/model/data/User.kt

```diff
@@ -5,6 +5,6 @@ data class User(
     val firstName: String,
     val lastName: String,
     val email: String,
-    var associations: List<String>,
+    var associations: List<Triple<String, String, Int>>,
```

Comment:
It was better before because we only store the IDs of the Association in the User, not all the information